common/scylla_cloud_io_setup

```diff
@@ -187,6 +187,11 @@ class aws_io_setup(cloud_io_setup):
             self.disk_properties["read_bandwidth"] = 1656354602 * nr_disks
             self.disk_properties["write_iops"] = 92233 * nr_disks
             self.disk_properties["write_bandwidth"] = 1028010325 * nr_disks
+        elif self.idata.instancetype == "i4i.xlarge":
+            self.disk_properties["read_iops"] = 109695 * nr_disks
```

Comment:
This number seems too low

Reply:
What about 2xlarge, 4xlarge?

Reply:
@avikivity All instance types were added now

As for the numbers. i have locally ran `scylla_io_setup` on all instance types few times, and the results are almost the same

Reply:
@avikivity ping